<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>servoboard_main</class>
 <widget class="QWidget" name="servoboard_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>440</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
}

QTextEdit{
 	border-width: 1px; 
	border-style: solid; 
	border-radius: 5px;
	border-color: rgb(0, 0, 0);
}


QGroupBox{
 	border-width: 1px; 
	border-style: solid; 
	border-radius: 5px;
	border-color: rgb(0, 0, 0);

}


QSpinBox {
	background-color: rgb(255, 255, 255);
}

QSlider::groove:vertical {
border: 1px solid #bbb;
background: white;
height: 68px;
border-radius: 4px;
}

QSlider::sub-page:vertical {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 68px;
border-radius: 4px;
}

QSlider::add-page:vertical {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:vertical {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
height:10px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:vertical:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:vertical:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:vertical:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:vertical:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

QPushButton {
color: #222;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #BBB);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #fff);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="1" column="0">
    <widget class="QTextEdit" name="txtSequence"/>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridMainLayout">
     <item row="1" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnPlaySelected">
       <property name="toolTip">
        <string>Play all enabled servos to the serial port</string>
       </property>
       <property name="text">
        <string>Play Selected</string>
       </property>
       <property name="icon">
        <iconset resource="../SerialPortIcons.qrc">
         <normaloff>:/res/images/play_large_green.png</normaloff>:/res/images/play_large_green.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="7">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="lblServoBoardControl">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Servo Board Control</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_underline0">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacerTop">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="5">
      <widget class="QCheckBox" name="chkUseAdvanced">
       <property name="toolTip">
        <string>Store advanced options with the current position</string>
       </property>
       <property name="text">
        <string>Use Advanced Options</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnClearAll">
       <property name="toolTip">
        <string>Unenable all servos</string>
       </property>
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnPause">
       <property name="toolTip">
        <string>Pause playback of the current sequence</string>
       </property>
       <property name="text">
        <string>Pause Sequence</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnStopSequence">
       <property name="toolTip">
        <string>Stop playback of the current sequence</string>
       </property>
       <property name="text">
        <string>Stop Sequence</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnPlaySequence">
       <property name="toolTip">
        <string>Begin playback of current sequence</string>
       </property>
       <property name="text">
        <string>Play Sequence</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnAdvancedLineOptions">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Set advanced options about the current position</string>
       </property>
       <property name="text">
        <string>Advanced Line</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnSelectAll">
       <property name="toolTip">
        <string>Enable all servos</string>
       </property>
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="btnStore">
       <property name="toolTip">
        <string>Store position as next one in the sequence</string>
       </property>
       <property name="text">
        <string>Store Position</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnSetStartPosition">
       <property name="toolTip">
        <string>Choose the intial position to be burnt to the board</string>
       </property>
       <property name="text">
        <string>Set Start Position</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="btnHotModeStartStop">
       <property name="toolTip">
        <string>DANERGOUS!! Trasmit all positions to the board constantly</string>
       </property>
       <property name="text">
        <string>We're Hot</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="btnLoadCurrent">
       <property name="toolTip">
        <string>Move current position of servos to sliders</string>
       </property>
       <property name="text">
        <string>Load Current Position</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../SerialPortIcons.qrc"/>
 </resources>
 <connections/>
</ui>
